x-pack/plugins/security_solution/cypress/e2e/detection_rules/prebuilt_rules_install_update_workflows.cy.ts

```diff
@@ -0,0 +1,197 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { BulkInstallPackageInfo } from '@kbn/fleet-plugin/common';
+import type { Rule } from '../../../public/detection_engine/rule_management/logic/types';
+import { createRuleAssetSavedObject } from '../../helpers/rules';
+import {
+  ELASTIC_RULES_BTN,
+  LOAD_PREBUILT_RULES_BTN,
+  LOAD_PREBUILT_RULES_ON_PAGE_HEADER_BTN,
+  RULES_MANAGEMENT_TABLE,
+  RULES_ROW,
+  TOASTER,
+  UPDATE_PREBUILT_RULES_CALLOUT_BUTTON,
+} from '../../screens/alerts_detection_rules';
+import { waitForRulesTableToBeLoaded } from '../../tasks/alerts_detection_rules';
+import {
+  createNewRuleAsset,
+  getRuleAssets,
+  installAvailableRules,
+  preventPrebuiltRulesPackageInstallation,
+} from '../../tasks/api_calls/prebuilt_rules';
+import { cleanKibana, resetRulesTableState, deleteAlertsAndRules } from '../../tasks/common';
+import { esArchiverResetKibana } from '../../tasks/es_archiver';
+import { login, visitWithoutDateRange } from '../../tasks/login';
+import { SECURITY_DETECTIONS_RULES_URL } from '../../urls/navigation';
+
+describe('Detection rules, Prebuilt Rules Installation and Update workflow', () => {
+  before(() => {
+    cleanKibana();
```

Comment:
As with the `esArchiverResetKibana()` we are setting a clean instance, I believe there is no need of having `cleanKibana`, which would also help to speed up the test execution.